singularity-opac/neutrinos/mean_s_opacity_neutrinos.hpp

```diff
@@ -0,0 +1,234 @@
+// ======================================================================
+// © 2022. Triad National Security, LLC. All rights reserved.  This
+// program was produced under U.S. Government contract
+// 89233218CNA000001 for Los Alamos National Laboratory (LANL), which
+// is operated by Triad National Security, LLC for the U.S.
+// Department of Energy/National Nuclear Security Administration. All
+// rights in the program are reserved by Triad National Security, LLC,
+// and the U.S. Department of Energy/National Nuclear Security
+// Administration. The Government is granted for itself and others
+// acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+// license in this material to reproduce, prepare derivative works,
+// distribute copies to the public, perform publicly and display
+// publicly, and to permit others to do so.
+// ======================================================================
+
+#ifndef SINGULARITY_OPAC_NEUTRINOS_MEAN_S_OPACITY_NEUTRINOS_
+#define SINGULARITY_OPAC_NEUTRINOS_MEAN_S_OPACITY_NEUTRINOS_
+
+#include <cmath>
+
+#include <ports-of-call/portability.hpp>
+#include <singularity-opac/base/radiation_types.hpp>
+#include <singularity-opac/base/sp5.hpp>
+#include <singularity-opac/constants/constants.hpp>
+#include <spiner/databox.hpp>
+
+#include <singularity-opac/neutrinos/mean_neutrino_s_variant.hpp>
+#include <singularity-opac/neutrinos/thermal_distributions_neutrinos.hpp>
+
+namespace singularity {
+namespace neutrinos {
+namespace impl {
+
+#define EPS (10.0 * std::numeric_limits<Real>::min())
+
+// TODO(BRR) Note: It is assumed that lambda is constant for all densities,
```

Comment:
Not sure I understand this comment

Reply:
We provide only one `Real *lambda` to the mean opacity constructor and these values in `lambda` are the same for every opacity call as we build up our table/integrals over density, temperature, and Ye.